src/pcap/api.go

```diff
@@ -0,0 +1,650 @@
+package pcap
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"sync"
+	"time"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/status"
+)
+
+const concurrentCapturesPerClient = 10
+
+type API struct {
+	bufConf  BufferConf
+	handlers map[string]CaptureHandler
+	agents   AgentMTLS
+	UnimplementedAPIServer
```

Comment:
Please separate composition and field definitions from another.

Reply:
done